fixtures/showtec/club-par-12-6-rgbwauv.json

```
@@ -0,0 +1,201 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Club Par 12/6 RGBWAUV",
+  "categories": ["Color Changer"],
+  "meta": {
+    "authors": ["Tobias"],
+    "createDate": "2022-11-03",
+    "lastModifyDate": "2022-11-03"
+  },
+  "links": {
+    "manual": [
+      "https://www.highlite.com/de/mwdownloads/download/link/id/17430150/",
+      "https://www.highlite.com/de/mwdownloads/download/link/id/17430148/"
```

The English manual is enough; other languages can often be found via the product page.

```suggestion
      "https://www.highlite.com/en/mwdownloads/download/link/id/17430150/"
```